capstone/src/main/java/com/google/sps/search/servlets/NameTestData.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.search.servlets;
+
+import java.util.List;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.sps.Objects.User;
+import com.google.sps.search.CsvReader;
+
+@WebServlet("/name-test-data")
+public class NameTestData extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) {
```

Comment:
If you do end up going the load-on-startup route, this would be changed to `init()`